coretex/entities/project/project.py

```diff
@@ -110,3 +125,34 @@ def addTask(self, name: str, description: Optional[str]) -> bool:
 
         self.tasks.append(task)
         return True
+
+    def updateVisibility(self, visibility: ProjectVisibility) -> bool:
```

Comment:
If a function raises an exception if it fails to change visibility, why does it return a bool?

Reply:
Function is updating the visibility in the databse, but locally fetched entity property does not get updated